bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+############################################################
+# Help                                                     #
+############################################################
+Help()
+{
+   # Display Help
+   echo "Bootstrap spack-stack with Miniconda installation."
+   echo
+   echo "options:"
+   echo "-p     spack-stack installation prefix"
+}
+
+prefix=""
+miniconda_ver="py39_4.12.0"
```

Comment:
Good to pin the version! Is that `python-3.9.7`?

Reply:
It's actually 3.9.12. They don't encode the minor version.